.travis.yml

```diff
@@ -5,7 +5,7 @@ addons:
   chrome: stable
 node_js:
   - "7"
-  - "node"
```

Comment:
We might as well remove line 8 instead of running this twice against Node 7. Running against Node latest was useful as it let us know that Node 11 will cause us some problems. But we don't want to block ourselves on that now. So let's remove line 8.